services/apps/src/apps/service/workspace.clj

```diff
@@ -1,9 +1,11 @@
 (ns apps.service.workspace
   (:use [korma.db :only [transaction]])
-  (:require [apps.persistence.workspace :as wp]))
+  (:require [apps.clients.iplant-groups :as iplant-groups]
+            [apps.persistence.workspace :as wp]))
 
 (defn get-workspace
-  [{:keys [username]}]
+  [{short-username :shortUsername :keys [username]}]
+  (iplant-groups/add-de-user short-username)
```

Comment:
OK, that makes sense.
We can leave it where it is for now, then add the new endpoint when we refactor the `bootstrap` endpoint.
So this PR looks good to me :shipit: